pecan/dbfiles/posterior/532/prior.distns.csv

```diff
@@ -0,0 +1,20 @@
+"distn","parama","paramb","n"
```

Comment:
There's a long series of posteriors added in pecan/dbfiles/posterior. If these are tests they should be in the correct place. If they are not tests they should be removed